src/app/components/msel-list/msel-list.component.html

```diff
@@ -133,7 +49,49 @@
       </button>
     </div>
   </div>
-
+<ng-container>
+  <div class="buttons-container">
+    <div class="button-items">
+      <button mat-icon-button (click)="addMsel(); $event.stopPropagation()" title="Add blank MSEL" [disabled]="!isContentDeveloper">

```

Comment:
Is the $event.stopPropagation() needed here? I don't see a default behavior that needs to be stopped.

Reply:
I tested it out without this function and it is working for me. However, Tim is the one who added it, so I would like to obtain his insight in case there's a specific reason on why it was added.

Reply:
It was most likely a cut and paste remnant.  It can be removed.  Good catch Andrew!